cmd/controller-manager/app/options/kubecloudshared.go

```diff
@@ -49,6 +50,7 @@ func (o *KubeCloudSharedOptions) AddFlags(fs *pflag.FlagSet) {
 		return
 	}
 
+	fs.SetOutput(logs.WarningWriter())
```

Comment:
Is everything a FlagSet logs suitable for the Warning level?

Reply:
@mtaufen i think so . it was using `os.Stderr` so seemed fair to use `Warning` for those

Reply:
`os.Stderr` could be used for warnings, errors, or fatal errors. Can you double-check?